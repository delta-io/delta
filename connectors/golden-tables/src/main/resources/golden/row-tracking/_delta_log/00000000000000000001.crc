{"txnId":"0f554c74-9a5e-4cf7-b1a9-a9f4d9ee4182","tableSizeBytes":696,"numFiles":1,"numMetadata":1,"numProtocol":1,"setTransactions":[],"domainMetadata":[{"domain":"delta.rowTracking","configuration":"{\"rowIdHighWaterMark\":4,\"domainName\":\"delta.rowTracking\"}","removed":false}],"metadata":{"id":"afea3664-cab2-42bd-88e1-3aa63a5ef4ee","format":{"provider":"parquet","options":{}},"schemaString":"{\"type\":\"struct\",\"fields\":[{\"name\":\"id\",\"type\":\"integer\",\"nullable\":true,\"metadata\":{}},{\"name\":\"value\",\"type\":\"string\",\"nullable\":true,\"metadata\":{}}]}","partitionColumns":[],"configuration":{"delta.enableRowTracking":"true","delta.rowTracking.materializedRowIdColumnName":"_row-id-col-d01a1e0e-fa6d-49b3-9259-dff5fad9bd75","delta.rowTracking.materializedRowCommitVersionColumnName":"_row-commit-version-col-3030e3a2-589d-4e5d-9dd9-7fc973c01ccb"},"createdTime":1752505895046},"protocol":{"minReaderVersion":1,"minWriterVersion":7,"writerFeatures":["domainMetadata","rowTracking","appendOnly","invariants"]},"allFiles":[{"path":"part-00000-ac602f34-1383-49b5-a18c-1d608ea58845-c000.snappy.parquet","partitionValues":{},"size":696,"modificationTime":1752505900286,"dataChange":false,"stats":"{\"numRecords\":5,\"minValues\":{\"id\":1,\"value\":\"A\"},\"maxValues\":{\"id\":5,\"value\":\"E\"},\"nullCount\":{\"id\":0,\"value\":0}}","baseRowId":0,"defaultRowCommitVersion":1}]}
