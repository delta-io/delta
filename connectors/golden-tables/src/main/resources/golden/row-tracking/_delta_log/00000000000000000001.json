{"commitInfo":{"timestamp":1752505900548,"operation":"WRITE","operationParameters":{"mode":"Overwrite","partitionBy":"[]"},"readVersion":0,"isolationLevel":"Serializable","isBlindAppend":false,"operationMetrics":{"numFiles":"1","numRemovedFiles":"0","numRemovedBytes":"0","numOutputRows":"5","numOutputBytes":"696"},"engineInfo":"Apache-Spark/3.5.3 Delta-Lake/3.4.0-SNAPSHOT","txnId":"0f554c74-9a5e-4cf7-b1a9-a9f4d9ee4182"}}
{"add":{"path":"part-00000-ac602f34-1383-49b5-a18c-1d608ea58845-c000.snappy.parquet","partitionValues":{},"size":696,"modificationTime":1752505900286,"dataChange":true,"stats":"{\"numRecords\":5,\"minValues\":{\"id\":1,\"value\":\"A\"},\"maxValues\":{\"id\":5,\"value\":\"E\"},\"nullCount\":{\"id\":0,\"value\":0}}","baseRowId":0,"defaultRowCommitVersion":1}}
{"domainMetadata":{"domain":"delta.rowTracking","configuration":"{\"rowIdHighWaterMark\":4,\"domainName\":\"delta.rowTracking\"}","removed":false}}
